### ISSUE TITLE
Mettre à jour la liste de démo utilisé lors de la création ou la modification d'une liste

### ISSUE THREAD

**Issue by patou** [patou/liste-envies#121]
Compléter et mettre à jour la liste des envies de démo utilise lors de la création ou la modification d'une liste.

- Certaines images ne s'affichent plus
- Avec des envies plus générique
- Ne pas afficher un email personnel (kie...anne@gmail.com)
- Dates des envies plus proche (4 ans, ça montre que c'est pas à jour).
- Avoir des envies qui change.

**Comment by manudss** [comment]
Oui, quand on aura une liste de démo, pour chaque type de liste, je voudrais mettre que quand on choisit une type de liste, cela charge, la liste de démo pour ce type.
